source/install-desktops/install-desktop-app.rst

```diff
@@ -0,0 +1,103 @@
+.. _install-desktops-install-desktop-app:
+
+Install Desktop App
+===================
+
+No we will go through installing the provided `bc_desktop`_ app as well as
+configuring it work at your center.
+
+#. We will build this app in our previous web applications setup directory
+   (discussed in :ref:`install-apps`):
+
+   .. code-block:: sh
+
+      cd ~/ood/src/apps/
+
+#. By default this app is not built and installed with all the other required
+   OOD apps. You will need to build this app explicitly with:
+
+   .. code-block:: sh
+
+      scl enable rh-ruby22 nodejs010 git19 -- rake build:bc_desktop
```

Comment:
We currently implement it on the OOD Demo server with:

```sh
scl enable rh-ruby22 nodejs010 git19 -- rake build:bc_desktop OOD_SITE=osc
```

also the build does more than a `git clone`, it can change the repository as well as `git fetch` and `git checkout`.